lib/ansible/module_utils/_internal/_no_six.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import inspect
+
```

Comment:
The `six` module needs to be imported normally so that AnsiballZ sees it:

```suggestion

from ansible.module_utils import six
```